examples/rmtchar/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# ##############################################################################
+# apps/examples/rmtchar/CMakeLists.txt
```

Comment:
look like rmt is an infrared remote control hardware? why not use lirc driver model?

https://github.com/apache/nuttx/blob/master/include/nuttx/lirc.h
https://github.com/apache/nuttx/blob/master/include/nuttx/rc/lirc_dev.h
https://github.com/apache/nuttx/tree/master/drivers/rc
https://github.com/apache/nuttx-apps/tree/master/testing/irtest
https://www.lirc.org/

Reply:
Yes, the `lirc` is intended to be used with the RMT peripheral, but we would like to provide an application to test the "raw" RMT character driver, just to be able to test the lower-half driver implementation. This is similar to the [`i2schar`](https://github.com/apache/nuttx-apps/tree/master/examples/i2schar) app ;)

On ESP32, for instance, the RMT peripheral is used as a generic one-wire peripheral that enables a lot of different protocols to be implemented.

I'll fix the issues on Monday!